notebooks/216-machine-translation/machine-translation-demo.ipynb

```diff
@@ -0,0 +1,304 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Machine translation demo\n",
+    "This demo utilizes Intel's pre-trained model that translates from English to German. More information about the model can be found [here](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/intel/machine-translation-nar-en-de-0002/README.md)."
```

Comment:
Please introduce a bit more about the model you used in this notebook, to let others know more about the model or any other possible model they could choose for completing the machine translation task from OMZ.